docs/howto/add-identify.md

```diff
@@ -1,12 +1,114 @@
+fortunately you can customize the look of the result by writing a custom template for the data.  Instead of an "auto"
+generated template, change the template to be:
+
+```xml
+        <layer name="county_borders" status="on">
+            <template name="identify"><![CDATA[
+            <div>
+                <div class="feature-class county_borders">
+                County
+                </div>
+                <div class="item">
+                    <label>Name:</label>{{properties.COUNTYNAME}}
+                </div>
+            </div>
+            ]]>
+            </template>
+        </layer>
+```
+
+In the example above, there are a few things to note:
+* Customizing the look of the results is done using HTML syntax
+* GeoMoose substitutes values for references it knows about when encounters `{{ }}`
+* Referencing the feature's value is done using `properties.COUNTYNAME`, where the `properties` part is from the GeoJSON
+format of the query results (so that is common to evert identify template), and the `COUNTYNAME` is the
+**case-sensitive** attribute name specific to the layer
+
+Because the template is HTML (and will be interpreted by the browser), you can do things like adding a link based on a
+feature's values:
+
+```xml
+        <layer name="county_borders" status="on">
+            <template name="identify"><![CDATA[
+            <div>
+                <div class="feature-class county_borders">
+                County
+                </div>
+                <div class="item">
+                    <label>Name:</label>
+                    <a href="http://www.census.gov/quickfacts/table/PST045215/{{properties.FIPS}},{{properties.COUNTYFIPS}}" target="_blank">
+                        {{properties.COUNTYNAME}}
+                    </a>
+                </div>
+            </div>
+            ]]>
+            </template>
+        </layer>
+```
+
+And since it is interpreted by the browser, it can even contain JavaScript:
+
+```xml
+            <template name="identify"><![CDATA[
+            <div>
+                <div class="feature-class county_borders">
+                County
+                </div>
+                <div class="item">
+                    <label>Name:</label>
+                    <a href=""
+                            onClick="window.open( 'http://www.census.gov/quickfacts/table/PST045215/{{properties.FIPS}},{{properties.COUNTYFIPS}}', 'Details','width=600,height=1000' ); return false"
+                            target="_blank">
+                        {{properties.COUNTYNAME}}</a>
+                </div>
+            </div>
+            ]]>
+            </template>
+```
 
 ## The Firestations layer
 
-This how-to references the `firestations` source and `fire_stations` layer.
+This section references the `firestations` source and `fire_stations` layer.
 For more information on setting those up in your local demo, read the
 [How-to add a layer](./add-a-layer.md) guide.
```

Comment:
For consistency with the index: `the [Adding a layer](./add-a-layer.md) guide`

Did I mention that I hate markdown? I would have expected rubyists to have picked something that wasn't so actively hostile to DRY.

Reply:
Good point.  My intent with this PR was just to establish the top level layout and at least list some more sections so others could contribute.  All of the existing files need review and editing.  Huge task; just getting started...